extension/js/common/core/crypto/key.ts

```diff
@@ -61,6 +61,15 @@ export type Contact = {
   revoked: boolean;
 };
 
+export class ContactUtil {
+
+public static isExpired = (contact: Contact) => {
+  return contact.pubkey && ((contact.expiresOn || Infinity) <= Date.now()
```

Comment:
this isn't necessary.
As `Contact` isn't stored in the database, its `expiredOn` attribute is calculated from its pubkey in `ContactStore.getKeyAttributes`. Anyway, the `Contact` class will probably be not required for this feature.